packages/eslint-plugin-calcite-components/README.md

```diff
@@ -49,6 +49,10 @@ This rule helps prevent usage of specific events and allows suggesting alternati
 
 This rule catches props/attributes that should be in the encapsulated HTML structure and not on the host element.
 
+- [`@esri/calcite-components/enforce-ref-last-prop`](./docs/enforce-ref-last-prop.md)
+
+This rule ensures the `ref` attribute is ordered last in a JSXElement to keep it up-to-date.
```

Comment:
Can you add info from https://github.com/Esri/calcite-design-system/pull/6530 to provide more context? The rule description and its doc file could also benefit from this extra info.